Assets/Ros2ForUnity/Plugins/Linux/x86_64/metadata_ros2cs.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" ?>
 <ros2cs>
-   <ros2>galactic</ros2>
+   <ros2>humble</ros2>
    <version>
-      <sha>0a6e83356d917c1c8b240e6c80725717e307da5f</sha>
-      <desc>0a6e833</desc>
-      <date>2022-05-27 10:59:43 +0200</date>
+      <sha>7e3d179ebf026a4999aa47d7753e1c85b66d5574</sha>
+      <desc>1.2.0</desc>
+      <date>2022-06-30 01:43:11 +0200</date>
    </version>
-   <standalone>1</standalone>
+   <standalone>0</standalone>
```

Comment:
The branch works on my Ubuntu 22 laptop with ROS humble. But please note, that non-standalone binaries are used, so you have to source ROS before running AWSIM.